lib/google_crawler/search/keyword_search_form.ex

```diff
@@ -0,0 +1,14 @@
+defmodule GoogleCrawler.Search.KeywordSearchForm do
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  embedded_schema do
+    field :term, :string
+    field :min_links_count, :integer
+    field :max_links_count, :integer
+  end
+
+  def changeset(keyword_search_form, attrs) do
```

Comment:
How about set default params for form?
```suggestion
  def changeset(keyword_search_form \\ %__MODULE__{}, attrs) do
```

So we can use it like this
```elixir
KeywordSearchForm.changeset(params["filter"] || %{})
```